<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1279</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #a39289;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="Cong_com">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #f7f7f7;
                      border:1px solid black;
                      color: black;
                      padding: 10px 30px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
   
                      border-radius: 0%;
                      min-width: 50px;
                      min-height: 50px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #8fb6cc;
                      border: none;
                      color: black;
                      padding: 10px 30px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;

                      border-radius: 0%;
                      min-width: 50px;
                      min-height: 50px;</string>
    </property>
    <property name="text">
     <string>Kết Nối</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TongDiem">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>160</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #3f4f54;
                      border:1px solid black;
                      color: white;
                      padding: 0px 0px;
                      text-align: left;
                      text-decoration: none;
                      font-size: 20px;
                      margin: 4px 2px;
                    
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string>Điểm:</string>
    </property>
   </widget>
   <widget class="QSlider" name="Toc_do">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>30</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <number>120</number>
    </property>
    <property name="maximum">
     <number>250</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>240</number>
    </property>
    <property name="sliderPosition">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                   border: 2px solid #cc4466;
                    background-color: #14172b;
                      border: none;
                      color: white;
                      padding: 10px 20px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 20px;
                      margin: 4px 2px;
  
                      border-radius: 50%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Tốc độ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Xoa_diem">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>380</y>
      <width>161</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                    
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
    </property>
    <property name="text">
     <string>Xóa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Them_diem">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>220</y>
      <width>161</width>
      <height>68</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                  
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
    </property>
    <property name="text">
     <string>Thêm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bat_dau">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;

                      font-size: 25px;
                      margin: 4px 2px;
                    
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
    </property>
    <property name="text">
     <string> Bắt Đầu</string>
    </property>
   </widget>
   <widget class="QLabel" name="x">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>580</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #f7f7f7;
 color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="y">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>650</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #f7f7f7;
 color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="z">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>720</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #f7f7f7;
 color: black;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Home">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #368aa8;
color: black;</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>431</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="j1cong">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                   
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>J1+</string>
     </property>
    </widget>
    <widget class="QLabel" name="Toado_j1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #2fb58f;
                      border:1px solid black;
                      color: #612f16;
                      padding: 10px 30px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 22px;
                      margin: 4px 2px;
                     
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="Toado_j3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #2fb58f;
                      border:1px solid black;
                      color: #612f16;
                      padding: 10px 30px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 22px;
                      margin: 4px 2px;
                    
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
     </property>
     <property name="text">
      <string>-120</string>
     </property>
    </widget>
    <widget class="QPushButton" name="j1tru">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                   
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>J1-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="j2cong">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                  
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>J2+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="j2tru">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                   
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>J2-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="j3cong">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
   
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>J3+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="j3tru">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                    
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>j3-</string>
     </property>
    </widget>
    <widget class="QLabel" name="Toado_j2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #2fb58f;
                      border:1px solid black;
                      color: #612f16;
                      padding: 10px 30px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 22px;
                      margin: 4px 2px;
                     
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
     </property>
     <property name="text">
      <string>53</string>
     </property>
    </widget>
    <widget class="QPushButton" name="daugap">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>111</width>
       <height>68</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #ebeff2;
                      border: none;
                      color: black;
                      padding: 15px 32px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 25px;
                      margin: 4px 2px;
                   
                      border-radius: 0%;
                      min-width: 30px;
                      min-height: 30px;</string>
     </property>
     <property name="text">
      <string>Kẹp</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Tong_diem">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #f7f7f7;
                      border:1px solid black;
                      color: black;
                      padding: 10px 30px;
                      text-align: center;
                      text-decoration: none;
                      display: inline-block;
                      font-size: 22px;
                      margin: 4px 2px;
                     
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>580</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;
                    background-color: #db7289;
                      border:1px solid black;
                      color: white;
                      padding: 0px 20px;
                      text-align: center;
                      text-decoration: none;
                      font-size: 16px;
 
           
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="text">
     <string>Tọa độ E:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>650</y>
      <width>66</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #368aa8;
color: black;font: 16pt &quot;Segoe UI&quot;;
                    background-color: #368aa8;
                      border:1px solid black;
                      color: black;
                      padding: 10px 20px;
                      text-align: center;
                      text-decoration: none;
                      font-size: 22px;
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>720</y>
      <width>66</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #368aa8;
color: black;font: 16pt &quot;Segoe UI&quot;;
                    background-color: #368aa8;
                      border:1px solid black;
                      color: black;
                      padding: 10px 20px;
                      text-align: center;
                      text-decoration: none;
                      font-size: 22px;
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>580</y>
      <width>66</width>
      <height>42</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #368aa8;
color: black;font: 16pt &quot;Segoe UI&quot;;
                    background-color: #368aa8;
                      border:1px solid black;
                      color: black;
                      padding: 10px 20px;
                      text-align: center;
                      text-decoration: none;
                      font-size: 22px;
                      border-radius: 0%;
                      min-width: 20px;
                      min-height: 20px;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="anhRobot">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>571</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1279</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
